CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and [Keep a Changelog](http://keepachangelog.com/).
 ### Added
 
 - Added internal mechanism to handle version-dependent parameters of API modules ([#151])
+- Support version-dependent `deletetalk` parameter in `WikiPage::delete()` and `WikiFile::delete()` ([#152])
```

Comment:
Shouldn't there also be a similar changelog entry for the logout method? :thinking:

Reply:
Oh, right. Added.